packages/evm-adapter/src/events.ts

```diff
@@ -15,6 +17,25 @@ export type EthPaymentEvent = {
   transactionDbId: string;
 };
 
+export type SplitPaymentEvent = {
```

Comment:
SplitPaymentEvent can extend from PaymentEvent, that's nicer type

Reply:
OK